frontend/zc_messaging/src/pages/threads/index.jsx

```diff
@@ -1,13 +1,87 @@
+import { useEffect, useState } from "react"
 import { Helmet } from "react-helmet"
+import { ThreadList } from "../../component"
 import generatePageTitle from "../../utils/generatePageTitle"
+import { getThreadHandler } from "./threads.utils"
+import "./threads.css"
+import { Toast } from "react-bootstrap"
 
 const Threads = () => {
+  const [threadsData, setThreadsData] = useState([])
+  const [statusMsgA, setStatusMsgA] = useState(false)
```

Comment:
if there is one status and one one error then use that
they don't need to have A and B

Reply:
Okay, Sir. Changed it